draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -803,6 +803,35 @@ Furthermore the hash algorithms MD2, MD4, MD5, RIPEMD-160, and SHA-1
 revealed fundamental weaknesses and they MUST NOT be used.
 
 ## Holder Binding {#holder_binding_security}
+Holder binding aims to ensure that the presenter of a credential is
+actually the legitimate Holder of the credential. There are, in general,
+two approaches to Holder Binding: Claims-based Holder Binding and
+Crpytographic Holder Binding.
+
+Claims-based Holder Binding means that the Issuer includes claims in the
+SD-JWT that a Verifier can correlate with the Holder, potentially with
+the help of other credentials presented at the same time. For example,
+in a vaccination certificate, the Issuer can include a claim that
+contains the Holder's name and birthdate, and a Verifier can correlate
+this data with the Holder's passport that has to be presented together
+with the vaccination certificate - either as a digital credential or a
+physical document. Biometric data may also be used for the purpose of
+binding a credential to a Holder.
```

Comment:
I think how biometrics can be used needs to be elaborated. I am still not sure if biometrics-based binding is sub-category of claim-based binding or if it is a category on its own.

not to stop merging of this PR, could we open an issue to track biometrics-based binding and remove it for now from this PR?

```suggestion
physical document.
```